### PR TITLE
Ncnews 19

### PR DESCRIPTION
**Pull request by HazemFahad**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -49,20 +54,6 @@ describe("GET /api/articles/:article_id returns correct article", () => {
...
-        article_id: 3,
-        author: "icellusedkars",
-        body: "some gifs",
-        created_at: "2020-11-03T09:12:00.000Z",
-        title: "Eight pug gifs that remind me of mitch",
-        topic: "mitch",
-        votes: 0,
-      },
-    });
-  });
```
Comment (HeatherEver) [inline __tests__/app.test.js:65]: Is there a reason this test was deleted? It looks pretty good!

File: models/models.js [hunk 2]
```diff
@@ -44,3 +44,13 @@ exports.updateVotes = async (articleID, voteNum) => {
 
   return result.rows[0];
 };
+
+exports.getAllUsers = async () => {
+  const queryString = `
+  SELECT username FROM users
```
Comment (HeatherEver) [inline models/models.js:50]: Don't forget your semicolon after users

**Review by HeatherEver — COMMENTED** [review]
Looking really good just a couple of tiny changes to make

File: __tests__/app.test.js (hunk 1 above)
Comment (HazemFahad) [reply __tests__/app.test.js:65]: Rose told me too lool she says it looked too similar to the next one! It ran just fine however

File: models/models.js (hunk 2 above)
Comment (HazemFahad) [reply models/models.js:50]: Whoopsie!

**Pull request merged**